recipe/meta.yaml

```diff
@@ -20,12 +19,12 @@ requirements:
     - pip
     - pytest-runner
     - setuptools >=20.0
-    - wheel
+    - wheel !=0.37.1
   run:
     - python
-    - setuptools >=20.0
+    - setuptools !=62.6
```

Comment:
We do not need `setuptools` in `run` at all, see https://github.com/PyCQA/astroid/blob/14a0d657809ab5b42e7d416b686953508e6d8b91/pyproject.toml#L34-L38